test/Tests/src/Data/Bool_Spec.enso

```diff
@@ -0,0 +1,18 @@
+from Standard.Base import all
+
+import Standard.Test
+from Standard.Base.Data.Ordering import Equal, Less, Greater
+
+spec =
+    Test.group "Booleans" <|
+        Test.specify "should allow converting Bools to Text values" <|
+            True.to_text . should_equal "True"
+            False.to_text . should_equal "False"
+
+        Test.specify "should allow for comparing Bools" <|
+            True.compare_to True . should_equal Equal
+            False.compare_to False . should_equal Equal
+            True.compare_to False . should_equal Greater
+            False.compare_to True . should_equal Less
+
+main = Test.Suite.run_main here.spec
```

Comment:
no newlines at the end of source files – according to our style guide, there should e always a newline (GitHub is indicating it too: 
<img width="407" alt="image" src="https://user-images.githubusercontent.com/1623053/156793146-007d759d-3bfb-401b-b993-5e85e9ee0e8b.png">

Reply:
:+1: